source/ADIOS.cpp

```diff
@@ -25,10 +25,8 @@
 #include "engine/bp/BPFileReader.h"
 #include "engine/bp/BPFileWriter.h"
 
-#ifdef ADIOS_HAVE_DATAMAN // external dependencies
 #include "engine/dataman/DataManReader.h"
```

Comment:
Actually in this case everything can be put into libadios2.a now. I have had a discussion with @williamfgc in the afternoon. Once DataMan is integrated into ADIOS2, we don't need to build shared libs anymore, but we will still need to keep the flexibility of loading arbitrary dynamic libraries in the DataMan engine.